jersey-tck/pom.xml

```diff
@@ -281,7 +281,7 @@
                         </goals>
                         <configuration>
                             <workingDirectory>${project.build.directory}/glassfish6/glassfish/bin</workingDirectory>
-                            <executable>asadmin</executable>
+                            <executable>./asadmin</executable>
```

Comment:
I wonder why you replace an unstable solution (relative path) by *another* unstable solution (another relative path) instead of using a stable solution (absolute path)?

Reply:
Yeah, having an absolute path is probably better. I updated the PR accordingly.